lib/neo4j/migrations/runner.rb

```diff
@@ -0,0 +1,106 @@
+module Neo4j
+  module Migrations
+    class Runner
+      STATUS_TABLE_FORMAT = '%-10s %-20s %s'.freeze
+      SEPARATOR = '--------------------------------------------------'.freeze
+      FILE_MISSING = '**** file missing ****'.freeze
+      STATUS_TABLE_HEADER = ['Status'.freeze, 'Migration ID'.freeze, 'Migration Name'.freeze].freeze
+      UP_MESSAGE = 'up'.freeze
+      DOWN_MESSAGE = 'down'.freeze
+
+      def initialize
+        @up_versions = SchemaMigration.all.pluck(:migration_id)
```

Comment:
Probably a minor performance improvement, but if this were a `Set` then the `include?` calls would be faster.